src/DataProtection/DataProtection/src/Internal/DockerUtils.cs

```diff
@@ -11,13 +11,13 @@ namespace Microsoft.AspNetCore.DataProtection.Internal
 {
     internal static class DockerUtils
```

Comment:
As long as we're at it, rename to `ContainerUtils`?

Reply:
Yeah, I thought about that. I'll do it :)